### DISCUSSION TITLE
Units of artificial porous media generated by porespy

### DISCUSSION THREAD

**Discussion by javierpagalo** [PMEAL/porespy#436] [Q&A] [answered]
Hello
I would like to know, when I generate a porous media using `generators.blobs`, what's the dimension of every voxel? 1um, 1mm or what?

Waiting for your response.
Thank you.

**Comment by ma-sadeghi** [comment, marked as answer]
Hi @javierpagalo. They're dimensionless, i.e. 1 voxel per voxel! Depending on what you do with the image, the unit will appear later on. For instance, if you do network extraction, then you need to provide the unit of your image to the network extraction routine. PoreSpy's SNOW algorithm, for instance, takes `voxel_size` as an input (a voxel size of 1 indicates 1 meter per voxel):

https://porespy.org/modules/generated/porespy.networks.snow.html

**Reply by javierpagalo** [reply]
Thank you 
It solves my question
